content/en/docs/best-practices/general-principles/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: General Principles
+description: The general principles for your Istio deployments.
+weight: 1
+icon: best-practices
+---
+
+We've identified the following general principles to help you get the most out
+of your Istio deployments.
+
+## Make your deployment simple
```

Comment:
Can we just call this "Prefer simplicity" or something?

Although reading it, I'm not actually sure what it really means. What does it mean to prefer simplicity in deployment model? Can we be much more explicit here, and say that we should prefer using as few clusters as possible, as few networks as possible, and as few meshes as possible?

Reply:
@smawson when I think about multicluster, I think we have 6 dimensions - that can range from 1..n.  Clearly 6 dimensions to infinity is a really big number, and your proposal of using as few of each dimension lines up with our ideas around compressing dimensions into something that is ultimately manageable.  I'm not sure i have helped this conversation along, but for @rcaballeromx this information may be helpful.

Cheers
-steve

Reply:
@smawson 
I think that the attempt here is what we had described previously as "prefer the simplest deployment possible" or "deploy the fewest number of `things`". I think it's a good principle in general, but could probably be reworded to be more clear.

Reply:
I'm updating the heading to try to capture the feedback. I landed on:
```suggestion
## Prefer simplicity
```